src/_codelabs/index.md

```diff
@@ -6,20 +6,21 @@ permalink: /codelabs
 toc: false
 ---
 
-You may find the following codelabs useful:
+See the latest Dart codelabs on the Google Developers site:
 
-* [Avast, Ye Pirates: Build an Angular App]({{site.webdev}}/codelabs/ng2)
-  <p>Learn how to use the Angular framework with Dart while you build a pirate
-  badge generator that runs in any modern browser.</p>
+* **[Intro to Dart for Java Developers](https://codelabs.developers.google.com/codelabs/from-java-to-dart)**
```

Comment:
I'd remove the boldface in this page. (I used it on webdev because the existing descriptions had bold codelab names, followed by descriptions, and without the boldface the new codelabs were overwhelmed by the old ones.)